src/net/bdew/neiaddons/appeng/SetFakeSlotCommandHandler.java

```diff
@@ -25,6 +25,15 @@ public void handle(NBTTagCompound data, EntityPlayerMP player) {
         if ((cont != null) && AddonAppeng.clsBaseContainer.isInstance(cont)) {
             Slot slot = cont.getSlot(slotNum);
             if ((slot != null) && AddonAppeng.clsSlotFake.isInstance(slot) && SlotHelper.isSlotEnabled(slot)) {
+                ItemStack targetStack = slot.getStack();
+                if (null != targetStack) {
+                    if (stack.isItemEqual(targetStack)) {
+                        stack.stackSize = slot.getStack().stackSize + stack.stackSize;
+                        if (stack.stackSize > 127) { // add this check if no patch this in AE encoder
```

Comment:
Why 127 instead of 64? Is it that there are some handling for oversized itemstack inside AE2? @Usernm0

Reply:
127 items in item stack works well in ae2 patterns. 

[example screen](https://i.imgur.com/ObvFkzd.png)

We can't use more than 127 because itemStack signed byte type in minecraft. 
Check more than 127 and set to 127 to be sure not broke anything.

Reply:
All right then.